ui/app/models/kv/metadata.js

```diff
@@ -95,9 +95,14 @@ export default class KvSecretMetadataModel extends Model {
     };
   }
 
+  get permissionsPath() {
+    return this.fullSecretPath || this.path;
```

Comment:
Curious why `fullSecretPath` wouldn't always be the value to use if it represents the full path to the secret.

Reply:
Because it's generated by the serializer (not easy to see from the diff). And the payload does not always return data.keys on the payload. See [here](https://github.com/hashicorp/vault/blob/main/ui/app/serializers/kv/metadata.js#L22).

An example of when the serializer wouldn't return this.fullSecretPath is when we're looking at a secret's details.